services/auth/docs/Authorization.md

```diff
@@ -1,45 +1,45 @@
-Authorization
-=============
+# Authorization
```

Comment:
What's the reasoning for changing the method of creating titles? By changing it only here this file becomes inconsistant with the rest of the API.

Reply:
My mistake - was trying out a save-on-format plugin for something else and didn't realize that it also formatted markdown files. Fixed.